From 749a52c0b9f742a538c9d8eef9dd1042322f7749 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 30 Apr 2019 15:52:55 +0300
Subject: [PATCH 22/43] ucm-imx8m-mini: dts: Some small changes

1) Defibed gpioledgrp ping group.
2) Cahnged GPIO3_IO16 init pad value.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
index e1de79294430..10cd529b0874 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
@@ -28,6 +28,9 @@
 
 	leds {
 		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_gpio_led>;
+
 		heartbeat-led {
 			label = "Heartbeat";
 			gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
@@ -160,7 +163,7 @@
 		pinctrl_hog_1: hoggrp-1 {
 			fsl,pins = <
 				MX8MM_IOMUXC_GPIO1_IO10_GPIO1_IO10	0x19
-				MX8MM_IOMUXC_NAND_READY_B_GPIO3_IO16	0x190
+				MX8MM_IOMUXC_NAND_READY_B_GPIO3_IO16	0x19
 				/* BT */
 				MX8MM_IOMUXC_SD1_DATA4_GPIO2_IO6	0x19
 				MX8MM_IOMUXC_SD1_DATA5_GPIO2_IO7	0x19
@@ -187,6 +190,13 @@
 			>;
 		};
 
+		pinctrl_gpio_led: gpioledgrp {
+			fsl,pins = <
+				MX8MM_IOMUXC_GPIO1_IO12_GPIO1_IO12	0x19
+			>;
+		};
+
+
 		pinctrl_i2c1: i2c1grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_I2C1_SCL_I2C1_SCL			0x400001c3
-- 
2.11.0

